L32-node-channel-API.md

```diff
@@ -0,0 +1,103 @@
+Node.js Channel API
+----
+* Author(s): mlumish
+* Approver: wenbozhu
+* Status: Draft
+* Implemented in: Node.js
+* Last updated: 
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/VR0C7DiG7bs
+
+## Abstract
+
+Expose the `Channel` class in the Node API, along with APIs for overriding the channel created in the `Client` constructor.
+
+## Background
+
+In the past, some users have requested the ability to explicitly create channels and share them among clients. In addition, another library needs to be able to intercept some channel functionality.
+
+
+### Related Proposals: 
+* A list of proposals this proposal builds on or supersedes.
```

Comment:
I think you meant N/A here instead of a placeholder text ? :) Not sure what we've done in the past, but I don't think I've recalled this placeholder text like this.